pkg/visitors/remove.go

```diff
@@ -26,6 +30,37 @@ func (v *Remove) Run(f uefi.Firmware) error {
 	find := Find{
 		Predicate: v.Predicate,
 	}
+	if v.RemoveDxes {
```

Comment:
If you had a slice of things to remove, and that slice could be empty, then this could just be
for _, v := range stuffToRemove {

this bool feels a bit unneeded.

Reply:
But I admit I don't really get this code.

Reply:
it's worse than that. Matches is already the slice of things to remove. RemoveDxes is required to have logic so that we operate on the Dxe FV instead of the whole flash image. It's one area where forth would be useful. I cant get the cli to pass in the FV instead of the flash image, so I have to do it in Run().

Reply:
This is one case where I would agree with you that utk's cli interface has negatively affected the way I wrote this visitor.